### PR TITLE
Feature/travis config

### PR DESCRIPTION
**Pull request by no-reply**
Let's see if this fixes our build problems.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,3 +14,6 @@ addons:
   code_climate:
     repo_token: 9b8c58a49bdfba5582d6a766fbe198e3720158e5e65c2ae00b1667e5e1bb8769
 env: CI=yes
+before_install: 'gem install bundler -v "~> major.minor.patch"'
```
Comment (markbreedlove) [inline .travis.yml]: Is that literally supposed to be "major.minor.patch"?

**Comment by markbreedlove** [comment]
Looks good to me.  We did have an issue with Travis CI failing the 2.3.0 build, only to pass it when it was re-run. Does that cause us any concern?

Otherwise, it would be great to have the bundler version specification added from the other commit.

**Comment by no-reply** [comment]
It seems like the intermittent failure is time dependent.

I'd suggest the way forward is to add the 2.3 build, and dig deeper if/when we see more failures. That the build reveals a potential problem seems like a bad reason to avoid it. :)

**Comment by markbreedlove** [comment]
Sure, I agree. I was just thinking the addition of 2.3, which might be its own feature branch with its own testing phase, might be a separate concern. I could go either way on merging it now or waiting.

**Comment by no-reply** [comment]
I force pushed a change removing the 2.3 build.

**Pull request merged**